docs/docs/classic-ml/getting-started/logging-first-model/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
```

Comment:
http://localhost:3000/ml/getting-started/logging-first-model/ shows empty page
<img width="1396" alt="image" src="https://github.com/user-attachments/assets/529a6d7d-3b81-4134-8972-4e7e851fd3be" />

Reply:
That's odd. It doesn't for me. Although, after every time I switch branches, I invalidate the cache and restart the server to make sure I don't have caching issues with REACT